views/audit/big_query_audit_log_v2.sql

```diff
@@ -0,0 +1,596 @@
+          '$.jobInsertion.job.jobConfig.labels.querytype'),
+        JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+          '$.jobChange.job.jobConfig.labels.querytype')
+      ) AS querytype
+    FROM `namratashah-ctr-sandbox.new_sink.cloudaudit_googleapis_com_data_access`
+  ),
+  data_audit AS (
+    SELECT
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson,
+        '$.tableDataChange.insertedRowsCount') AS insertRowCount,
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson, 
+        '$.tableDataChange.deletedRowsCount') AS deleteRowCount,
+      JSON_EXTRACT_SCALAR(protopayload_auditlog.metadataJson, 
+        '$.tableDataChange.reason') AS tableDataChangeReason,
+      CONCAT(
+        SPLIT(
+          JSON_EXTRACT(protopayload_auditlog.metadataJson, '$.tableDataChange.jobName'),
+        "/")[OFFSET(1)],
+      ":",
+        SPLIT(
+          JSON_EXTRACT(protopayload_auditlog.metadataJson, 
+            '$.tableDataChange.jobName'),
+          "/")[OFFSET(3)]
+      ) AS data_jobid
+    FROM `namratashah-ctr-sandbox.new_sink.cloudaudit_googleapis_com_data_access`
+  ) /* Best practice is to use a partitioned table */
+SELECT
+  principalEmail,
+  callerIp,
+  serviceName,
+  methodName,
+  eventName,
+  tableDataChangeReason,
+  loadDestinationTable, 
+  loadKmsKeyName, 
+  loadcreateDisposition,
+  loadwriteDisposition,
+  loadschemaJson,
+  loadsourceUris,
+  loadsourceUrisTruncated,
+  loadschemaJsonTruncated,
+  query,
+  queryTruncated,
+  queryDestinationTable, 
+  queryKmsKeyName, 
+  querycreateDisposition,
+  querywriteDisposition,
+  queryschemaJson,
+  querysourceUris,
+  queryschemaJsonTruncated,
+  queryName,
+  queryPriority,
+  statementType,
+  extractdestinationUris,
+  extractdestinationUrisTruncated,
+  extract_projectid,
+  extract_datasetid,
+  extract_tableid,
+  tableCopysourceTables,
+  tableCopysourceTablesTruncated,
+  tableCopycreateDisposition,
+  tableCopywriteDisposition,
+  tableCopyproject_id,
+  tableCopydataset_id,
+  tableCopytable_id,
+  tableCopykmsKeyname,
+  projectId,
+  jobId,
+  querytype,
+  insertRowCount,
+  deleteRowCount,
+  errorCode,
+  errorMessage,
+  STRUCT(
+    EXTRACT(MINUTE FROM startTime) AS minuteOfDay,
+    EXTRACT(HOUR FROM startTime) AS hourOfDay,
+    EXTRACT(DAYOFWEEK FROM startTime) - 1 AS dayOfWeek,
+    EXTRACT(DAYOFYEAR FROM startTime) AS dayOfYear,
+    EXTRACT(WEEK FROM startTime) AS week,
+    EXTRACT(MONTH FROM startTime) AS month,
+    EXTRACT(QUARTER FROM startTime) AS quarter,
+    EXTRACT(YEAR FROM startTime) AS year
+  ) AS date,
+  createTime,
+  startTime,
+  endTime,
+  runtimeMs,
+  runtimeSecs,
+  tableCopy,
+  /* This code queries data specific to the Copy operation */ CONCAT(
+    tableCopydataset_id, '.', tableCopytable_id
+  ) AS tableCopyDestinationTableRelativePath,
+  CONCAT(tableCopyproject_id, '.', tableCopydataset_id, '.', tableCopytable_id) AS tableCopyDestinationTableAbsolutePath,
```

Comment:
Decide if you want the concat on one line or split like you did with tableCopyDestinationTableRelativePath